deployed-devices/rest/keys/create-key/output/create-key.json

```diff
@@ -0,0 +1,11 @@
+{
+  "sid": "KYXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "friendly_name": null,
+  "fleet_sid": "FLXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "account_sid": "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "device_sid": "THXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "secret": "abcdeghijklmnopqrstuvwxyz",
```

Comment:
nitpick: for secrets, we like to stick to a placeholder that is more clearly a needs-to-be-replaced placeholder, like 'your_secret'